test/e2e/utils.go

```diff
@@ -162,13 +162,21 @@ func deletePolicy(name string) {
 		return apierrors.IsNotFound(err)
 	}, 60*time.Second, 1*time.Second).Should(BeTrue(), fmt.Sprintf("Policy %s not deleted", name))
 
-	// Wait for enactments to be removed
+	// Wait for enactments to be removed calculate timeout taking into account
```

Comment:
Please include the full description in the commit message. In the PR description it looks all right